enhancements/kube-apiserver/stability-api-removal-notifications.md

```diff
@@ -54,6 +53,45 @@ Fire an `APIRemovedInNextReleaseInUse` alert when an API that will be removed in
 On EUS releases, also fire an `APIRemovedInNextEUSReleaseInUse` alert when an API that will be removed in the next
 extended support release is in use.
 
+Introduce a `DeprecatedAPIRequest` API to track users of deprecated APIs.
+
+Example:
+
+```yaml
+version: tbd/v1
+kind: DeprecatedAPIRequest
+meta-data:
+  name: 'flowschemas.flowcontrol.apiserver.k8s.io-v1alpha1'
+status:
+  removedRelease: '1.21'
```

Comment:
<bike-shed>Shouldn't this be an OpenShift release? and `removedInRelease` sounds better, no?</bike-shed>

Reply:
@sttts, no it is all based on underlying k8s release.

removedRelease matches the wording used in related annotations, metric labels, and http Warning headers.